clusters/build-clusters/02_cluster/cert-manager/cert-issuer_clusterissuer.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: cert-issuer
+spec:
+  acme:
+    email: openshift-ci-robot@redhat.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    #server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: cert-issuer-account-key
```

Comment:
Where does this come from?

Reply:
I googled it too yesterday.
It is created by cert-manager (we just need to tell a secret name to use).
https://github.com/jetstack/cert-manager/issues/1751

We actually do not need it.
(in the acme.sh, we do not use it)

Reply:
SGTM